### PR TITLE
update for newer elixir image on dockerhub

### PR DESCRIPTION
**Pull request by jerdog**
The convenience image we link them to has been updated to 1.11.4, but this example config is stuck on 1.6.5. The updates bring the Dockerfile over to match.

# Description
Bring over the dockerfile from dockerhub so that our doc matches what the convenience image config is

# Reasons
Our docs for Docker Layer Caching were out of sync with what we're actually linking to, with the potential to cause confusion.

### CODE REVIEW

File: jekyll/_cci2/docker-layer-caching.md [hunk 1]
```diff
@@ -117,56 +117,95 @@ Let's use the following Dockerfile to illustrate how Docker Layer Caching works.
...
+FROM elixir:1.11.4
 
-# Make apt non-interactive
+# make Apt non-interactive
 RUN echo 'APT::Get::Assume-Yes "true";' > /etc/apt/apt.conf.d/90circleci \
   && echo 'DPkg::Options "--force-confnew";' >> /etc/apt/apt.conf.d/90circleci
 
 ENV DEBIAN_FRONTEND=noninteractive
 
+# Debian Jessie is EOL'd and original repos don't work.
```
Comment (thomasr8) [inline jekyll/_cci2/docker-layer-caching.md:128]:
```suggestion
# Debian Jessie is EOL'd and original repos don not work.
```

**Review by thomasr8 — APPROVED** [review]
Looks good! Just one minor comment.

File: jekyll/_cci2/docker-layer-caching.md (hunk 1 above)
Comment (jerdog) [reply jekyll/_cci2/docker-layer-caching.md:128]: I'll edit it and remove the extra `n` @thomasr8

**Comment by jerdog** [comment]
@thomasr8 I updated the word usage - can merge it now

**Pull request merged**